### ISSUE TITLE
How to use `hypothesis.settings` with `schemathesis.from_pytest_fixture`

### ISSUE THREAD

**Issue by orsinium** [kiwicom/schemathesis#69]
Example from readme:

```python
from hypothesis import settings

@settings(max_examples=5)
def test_something(client, case):
    ...
```

My code:

```
schema = schemathesis.from_pytest_fixture("swagger_schema")

@hypothesis.settings(max_examples=1)
@schema.parametrize()
def test_swagger_endpoints(client, case):
    ...
```

On tests run:

```
.../site-packages/schemathesis/lazy.py:19: in test
    schema = get_schema(request, self.fixture_name, method, endpoint)
.../site-packages/hypothesis/_settings.py:253: in new_test
    "Using `@settings` on a test without `@given` is completely pointless."
E   hypothesis.errors.InvalidArgument: Using `@settings` on a test without `@given` is completely pointless.
```

**Comment by orsinium** [comment]
I think, it would be cool to allow to pass hypothesis' settings into `schema.parametrize`

**Comment by Stranger6667** [comment]
Hello @orsinium !

It will be a useful feature, indeed! :) Will work on it soon

**Comment by Stranger6667** [comment]
There is a way to make it work - put `@settings` closer to the test function than `parametrize`:

```
schema = schemathesis.from_pytest_fixture("swagger_schema")

@schema.parametrize()
@hypothesis.settings(max_examples=1)
def test_swagger_endpoints(client, case):
    ...
```

Please, let us know if this will work for you (it works locally in the schemathesis test suite).

I'll add more info later

**Comment by Stranger6667** [comment]
Fixed in the new `0.7.0` release :) feel free to reopen the issue if the problem persists
